src/pages/Dashboard/components/DashboardHeader.tsx

```diff
@@ -32,54 +61,152 @@ export const DashboardHeader = () => {
     [claimableRewardsFiatBalance, earnFiatBalance, portfolioTotalFiatBalance],
   )
   const borderColor = useColorModeValue('gray.100', 'gray.750')
+  const bgColor = useColorModeValue('white', 'blackAlpha.100')
+
+  useEffect(() => {
+    if (activeRef.current) {
+      activeRef.current.scrollIntoView({ behavior: 'smooth', block: 'center', inline: 'center' })
```

Comment:
reminder but not a required change that you can opt chain fn calls `activeRef?.current?.scrollIntoView?.(...)`